### PR TITLE
Changes fire axe cabinet open/close from a verb to an alt-click.

### PR DESCRIPTION
**Pull request by JMoldy**
## About The Pull Request
Straight and simple enough, allows people to open and close the fire axe cabinet with alt-clicking. 

## Why It's Good For The Game
Adds a bit more alt-clicking functionality, putting the fire axe cabinet in line with the fire extinguisher cabinet. Currently to close a PROPERLY opened fire axe cabinet requires right-clicking on it.

## Changelog
:cl:
add: Fire axe cabinet can now be opened and closed with alt-click.
/:cl:

### CODE REVIEW

**Comment by AnturK** [comment]
Remove the verb and add examine tip while you're at it.

**Comment by JMoldy** [comment]
Why remove the verb?

File: code/game/objects/structures/fireaxe.dm [hunk 1]
```diff
@@ -173,15 +173,14 @@
...
 	if(locked)
 		to_chat(usr, "<span class='warning'>The [name] won't budge!</span>")
 		return
 	else
 		open = !open
 		update_icon()
 		return
+
+/obj/structure/fireaxecabinet/AltClick(mob/user)
+	toggle_open()
```
Comment (kriskog) [inline code/game/objects/structures/fireaxe.dm]: missing newline

Comment (JMoldy) [reply code/game/objects/structures/fireaxe.dm]: fixed, thanks

**Comment by cacogen** [comment]
code of the ancients

File: code/game/objects/structures/fireaxe.dm [hunk 2]
```diff
@@ -173,15 +173,14 @@
...
-	set src in oview(1)
-
+/obj/structure/fireaxecabinet/proc/toggle_open()
 	if(locked)
 		to_chat(usr, "<span class='warning'>The [name] won't budge!</span>")
 		return
 	else
 		open = !open
 		update_icon()
 		return
```
Comment (ShizCalev) [inline code/game/objects/structures/fireaxe.dm]: trailing return

Comment (ShizCalev) [reply code/game/objects/structures/fireaxe.dm]: clean up make attack_hand() call the proc too please.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by nemvar** [comment]
Pretty much all alt-clicking functionality has a hint in the examine. Please add.

File: code/game/objects/structures/fireaxe.dm [hunk 3]
```diff
@@ -115,21 +112,13 @@
 			to_chat(user, "<span class='notice'>You take the fire axe from the [name].</span>")
 			src.add_fingerprint(user)
 			update_icon()
-			return
```
Comment (ShizCalev) [inline code/game/objects/structures/fireaxe.dm]: bad

File: code/game/objects/structures/fireaxe.dm [hunk 4]
```diff
@@ -53,13 +52,11 @@
 			var/obj/item/twohanded/fireaxe/F = I
 			if(F.wielded)
 				to_chat(user, "<span class='warning'>Unwield the [F.name] first.</span>")
-				return
 			if(!user.transferItemToLoc(F, src))
 				return
 			fireaxe = F
 			to_chat(user, "<span class='notice'>You place the [F.name] back in the [name].</span>")
 			update_icon()
-			return
```
Comment (ShizCalev) [inline code/game/objects/structures/fireaxe.dm]: no

File: code/game/objects/structures/fireaxe.dm [hunk 5]
```diff
@@ -53,13 +52,11 @@
 			var/obj/item/twohanded/fireaxe/F = I
 			if(F.wielded)
 				to_chat(user, "<span class='warning'>Unwield the [F.name] first.</span>")
-				return
```
Comment (ShizCalev) [inline code/game/objects/structures/fireaxe.dm]: no

File: code/game/objects/structures/fireaxe.dm [hunk 6]
```diff
@@ -42,7 +42,6 @@
 		var/obj/item/stack/sheet/glass/G = I
 		if(G.get_amount() < 2)
 			to_chat(user, "<span class='warning'>You need two glass sheets to fix [src]!</span>")
-			return
```
Comment (ShizCalev) [inline code/game/objects/structures/fireaxe.dm]: no

File: code/game/objects/structures/fireaxe.dm [hunk 7]
```diff
@@ -1,6 +1,6 @@
 /obj/structure/fireaxecabinet
 	name = "fire axe cabinet"
-	desc = "There is a small label that reads \"For Emergency use only\" along with details for safe use of the axe. As if."
+	desc = "There is a small label that reads \"For Emergency use only\" along with details for safe use of the axe. As if. Alt-click to open and close."
```
Comment (ShizCalev) [inline code/game/objects/structures/fireaxe.dm]: this isn't quite what we were looking for. look at this for an example.
https://github.com/tgstation/tgstation/blob/0b03db27bff11b2276ed0eb879360baa8bb96512/code/game/objects/items/devices/radio/intercom.dm#L28-L33

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**